backend/data/scrapers/genedFormattedRaw.json

```diff
@@ -0,0 +1,12305 @@
+{
+    "ACTL3142": {
```

Comment:
what is the point of this file compared to the pureRaw one? i think that one is more useful.

Reply:
oh forgot to delete this!